citations-tool/tool/src/webapp/js/jquery.googlebooks.thumbnails.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * JQuery Plugin to get thumbnail images from Google Books
+ *
+ *  Given an img (or list of img's), which is representing a thumbnail for a media type,
+ *  that has a class of 'citationThumbnail' and an id containing its ISBN,
+ *  this plugin populates the src of that img with a small thumbnail from the Google Books API.
+ */
+(function ($) {
+
+    var GOOGLE_BOOKS_SEARCH_API = 'https://www.googleapis.com/books/v1/volumes?';
+
+    $.fn.setGoogleBooksThumbnailURL = function(element) {
+        if (element.id!=''){
+            $.getJSON(GOOGLE_BOOKS_SEARCH_API + 'q=isbn:' + element.id, function(data){
```

Comment:
Does the API return nicely cachable results? If so the browser will handle all the caching, otherwise it may be worth having an in-page simple cache so that if the same book is used throughout the course we don't end up making multiple calls the google.